config.yaml

```diff
@@ -0,0 +1,33 @@
+options:
+  log-level:
+    description: |
+      Configures the log level. 
+
+      Acceptable values are: "info", "debug", "warning", "error" and "critical"
+    default: "info"
+    type: string
+  s3-access-key:
```

Comment:
The guidelines said to use underscore instead of hyphen: `s3-access-key` -> `s3_access_key`